src/test/scala/com/itechart/ny_accidents/utils/DateUtilsTest.scala

```diff
@@ -1,17 +1,33 @@
 package com.itechart.ny_accidents.utils
 
-import com.itechart.ny_accidents.constants.GeneralConstants
+import java.time.format.DateTimeFormatter
+
 import org.scalatest.FunSpec
 
 class DateUtilsTest extends FunSpec {
   // Will be rewrite. Test failed on different timezone machines
   // TODO need fix
   describe("DateUtils") {
-//    it("should return valid value") {
-//      val expectedResult = 1558095780000L
-//      val testValue = "17.05.2019 15:23"
-//
-//      assert(DateUtils.parseDateToMillis(testValue, GeneralConstants.DATE_TIME_WEATHER_PATTERN).contains(expectedResult))
-//    }
+    //    it("should return valid value") {
```

Comment:
Remove commented code or enable and fix it

Reply:
Removed